packages/web/src/WidgetApp.ce.vue

```diff
@@ -443,6 +444,10 @@ onBeforeMount(() => {
 onMounted(async() => {
   // cf: https://stackoverflow.com/questions/69495211/vue3-route-query-empty
   // console.log('WidgetApp > onMounted > set router...')
+  if (Widget.is) {
```

Comment:
question : un truc que je pige pas... si `Widget.is === false` on skip donc ce qu'il y a ensuite, c'est-à-dire `await router.isready()`, `nav.setRouter(router)` etc...

Du coup si on est en mode "widget", le widget ne peut donc plus faire appel au router (ou du moins celui-ci est dégradé car il n'a pas pu passer par `nav.setRouter(router)` ? Si c'est bien le cas je pense que c'est un gros _drawback_ car on downgrade la capacité d'utilisation en "widget"...

Reply:
note : en fait si le widget utilise / utilisait bien le router auparavant, cette modif conduit bien à un _downgrade_ de cette partie. Je ne vais pas continuer à batailler pour tenter de défendre cet feat. et cet usage davantage. J'ai bien conscience et l'intention de développer en fonction des usages, mais cette partie (cohérence de feats entre widget et app) me semblait assez fondamentale en terme de postulats initiaux pour le projet. J'entends que tu as ta propre roadmap en tête et des arguments pour, donc je lâche l'affaire.

Reply:
Je suis d'accord que c'est effectivement un `downgrade` ! Et je suis aussi d'accord pour remettre le router quand l'on aura besoin du widget et des besoins spécifique à ce moment....

Merci pour l'approve !!